eos/observable.cc

```diff
@@ -611,17 +611,33 @@ namespace eos
                             &BToDLeptonNeutrino::differential_branching_ratio,
                             std::make_tuple("s")),
 
+            make_observable("B->Dlnu::normdBR/ds",
+                            &BToDLeptonNeutrino::normalized_differential_branching_ratio,
+                            std::make_tuple("s")),
+
             make_observable("B->Dlnu::BR",
                             &BToDLeptonNeutrino::integrated_branching_ratio,
                             std::make_tuple("s_min", "s_max")),
 
+            make_observable("B->Dlnu::normBR",
+                            &BToDLeptonNeutrino::normalized_integrated_branching_ratio,
+                            std::make_tuple("s_min", "s_max")),
+
             make_observable("B->Dlnu::R_D(s)",
                             &BToDLeptonNeutrino::differential_r_d,
                             std::make_tuple("s")),
 
             make_observable("B->Dlnu::R_D",
```

Comment:
Maybe you can use the new way of creating an observable that is actually
a ratio of two observables. Danny has pushed somewhere an example,
I think it is for R_Dstar(s).

Reply:
We have a separate issue for this, which I assigned to @gubernari already :-)

Reply:
Ok